ixwebsocket/IXNetSystem.cpp

```diff
@@ -59,7 +59,14 @@ namespace ix
 
```

Comment:
Great, the change is local then which makes this easier to maintain like you pointed out.